servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpRequest.java

```diff
@@ -145,6 +145,18 @@ public HttpRequest setQueryParameters(final String key, final String... values)
         return this;
     }
 
+    @Nullable
+    @Override
+    public String fragment() {
```

Comment:
Reminder to remove this override from all classes, except `AbstractDelegatingHttpRequest` and `DefaultHttpRequestMetaData`